docs/community/content/contribute/code-conduct.cn.md

```diff
@@ -19,74 +19,74 @@ chapter = true
+ - 除了用于继承的抽象类之外，尽量将类设计为 `final`。
  - 嵌套循环尽量提成方法。
  - 成员变量定义顺序以及参数传递顺序在各个类和方法中保持一致。
  - 优先使用卫语句。
  - 类和方法的访问权限控制为最小。
  - 方法所用到的私有方法应紧跟该方法，如果有多个私有方法，书写私有方法应与私有方法在原方法的出现顺序相同。
- - 方法入参和返回值不允许为`null`。
- - 优先使用三目运算符代替if else的返回和赋值语句。
- - 优先使用lombok代替构造器，getter, setter方法和log变量。
- - 优先考虑使用`LinkedList`，只有在需要通过下标获取集合中元素值时再使用`ArrayList`。
- - `ArrayList`，`HashMap`等可能产生扩容的集合类型必须指定集合初始大小，避免扩容。
+ - 方法入参和返回值不允许为 `null`。
+ - 优先使用三目运算符代替 if else 的返回和赋值语句。
+ - 优先使用 lombok 代替构造器，getter, setter 方法和 log 变量。
+ - 优先考虑使用 `LinkedList`，只有在需要通过下标获取集合中元素值时再使用 `ArrayList`。
+ - `ArrayList`，`HashMap` 等可能产生扩容的集合类型必须指定集合初始大小，避免扩容。
  - 日志与注释一律使用英文。
- - 注释只能包含javadoc，todo和fixme。
- - 公开的类和方法必须有javadoc，其他类和方法以及覆盖自父类的方法无需javadoc。
+ - 注释只能包含 javadoc，todo 和 fixme。
+ - 公开的类和方法必须有 javadoc，其他类和方法以及覆盖自父类的方法无需 javadoc。
  - 条件运算符（<表达式1> ? <表达式2> : <表达式3>）禁止`嵌套使用`
 
 ## 单元测试规范
 
  - 测试代码和生产代码需遵守相同代码规范。
- - 单元测试需遵循AIR（Automatic, Independent, Repeatable）设计理念。
-   - 自动化（Automatic）：单元测试应全自动执行，而非交互式。禁止人工检查输出结果，不允许使用`System.out`，`log`等，必须使用断言进行验证。
+ - 单元测试需遵循 AIR（Automatic, Independent, Repeatable）设计理念。
+   - 自动化（Automatic）：单元测试应全自动执行，而非交互式。禁止人工检查输出结果，不允许使用 `System.out`，`log` 等，必须使用断言进行验证。
    - 独立性（Independent）：禁止单元测试用例间的互相调用，禁止依赖执行的先后次序。每个单元测试均可独立运行。
    - 可重复执行（Repeatable）：单元测试不能受到外界环境的影响，可以重复执行。
- - 单元测试需遵循BCDE（Border, Correct, Design, Error）设计原则。
+ - 单元测试需遵循 BCDE（Border, Correct, Design, Error）设计原则。
    - 边界值测试（Border）：通过循环边界、特殊数值、数据顺序等边界的输入，得到预期结果。
    - 正确性测试（Correct）：通过正确的输入，得到预期结果。
    - 合理性设计（Design）：与生产代码设计相结合，设计高质量的单元测试。
    - 容错性测试（Error）：通过非法数据、异常流程等错误的输入，得到预期结果。
  - 如无特殊理由，测试需全覆盖。
  - 每个测试用例需精确断言。
  - 准备环境的代码和测试代码分离。
- - 只有junit `Assert`，hamcrest `CoreMatchers`，Mockito相关可以使用static import。
- - 单数据断言，应使用`assertTrue`，`assertFalse`，`assertNull`和`assertNotNull`。
- - 多数据断言，应使用`assertThat`。
- - 精确断言，尽量不使用`not`，`containsString`断言。
- - 测试用例的真实值应名为为actualXXX，期望值应命名为expectedXXX。
- - 测试类和`@Test`标注的方法无需javadoc。
+ - 只有 junit `Assert`，hamcrest `CoreMatchers`，Mockito 相关可以使用 static import。
+ - 单数据断言，应使用 `assertTrue`，`assertFalse`，`assertNull` 和 `assertNotNull`。
+ - 多数据断言，应使用 `assertThat`。
+ - 精确断言，尽量不使用 `not`，`containsString` 断言。
+ - 测试用例的真实值应名为为 actual XXX，期望值应命名为 expected XXX。
+ - 测试类和 `@Test` 标注的方法无需 javadoc。
 
-## G4编码规范
+## G4 编码规范
  - 公共规范
-   - 每行长度不超过`200`个字符，保证每一行语义完整以便于理解。
+   - 每行长度不超过 `200` 个字符，保证每一行语义完整以便于理解。
  - 词法解析规范
    - 每个规则一行，规则间无需空行。
-   - 规则名称使用大写字母。如果名称由多个单词组成，用`下划线`间隔。`DataType`和`Symbol`的规则命名以`下划线`结尾。与ANTLR内置变量或关键字重名的规则在结尾加`下划线`以示区分。
-   - 不对外暴露的规则使用`fragment`，`fragment`定义的规则需在其服务的规则之后声明。
-   - 公用规则定义放在`Keyword.g4`，每个数据库可以有自己特有的规则定义。例如：`MySQLKeyword.g4`。
+   - 规则名称使用大写字母。如果名称由多个单词组成，用`下划线`间隔。`DataType` 和 `Symbol` 的规则命名以 `下划线` 结尾。与 ANTLR 内置变量或关键字重名的规则在结尾加`下划线`以示区分。
+   - 不对外暴露的规则使用 `fragment`，`fragment` 定义的规则需在其服务的规则之后声明。
+   - 公用规则定义放在 `Keyword.g4`，每个数据库可以有自己特有的规则定义。例如：`MySQLKeyword.g4`。
  - 语法解析规范
    - 每个规则结束后空一行，空行无需缩进。
    - 规则名称前面不空格，`冒号`后空一格再开始写规则，`分号`在单独一行并保持和上一行相同缩进。
-   - 如果一个规则的分支超过`5`个，则每个分支一行。
+   - 如果一个规则的分支超过 `5` 个，则每个分支一行。
    - 规则命名采用java变量的驼峰形式。
```

Comment:
The last 2 lines may be revised together.

Reply:
Thank you for letting me know, I corrected it as well.

Reply:
Can you confirm if its visible to you as well ?

Reply:
@jhanvi831 perfect, thanks for your contribution.